Sources/Datadog/RUM/DataModels/RUMDataModelsMapping.swift

```diff
@@ -0,0 +1,40 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+/* Collection of mappings from SDK models to `RUMDataModel` format. */
+
+internal extension RUMUUID {
+    var toRUMDataFormat: String {
+        return rawValue.uuidString.lowercased()
+    }
+}
+
+internal extension RUMErrorSource {
+    var toRUMDataFormat: RUMSource {
+        switch self {
+        case .source: return .source
+        case .console: return .console
+        case .network: return .network
+        case .agent: return .agent
+        case .logger: return .logger
+        case .webview: return .webview
+        }
+    }
+}
+
+internal extension RUMUserActionType {
+    /// TODO: RUMM-517 Map `RUMUserActionType` to enum cases from generated models
+    var toRUMDataFormat: String {
+        switch self {
+        case .tap: return "tap"
+        case .scroll: return "scroll"
+        case .swipe: return "swipe"
+        case .custom: return "custom"
+        }
+    }
+}
```

Comment:
`RUMDataModels` belong to the transport layer. This mapping is to separate domains and translate user-layer (public) types into their DTOs.

Removing this separation will not only tangle domains together, but also:
* make public API stability depend on internal models format,
* introduce more effort in `RUMDataModels` generation, as some names are not user friendly (like `RUMMethod.methodGET` instead of simply `.GET`),
* expose unnecessary things in public (`RUMDataModels` are `Codable` and some are also `Equatable`), forcing us to constantly consider this in development.

This mapping is manual, but straightforward. Helps a lot and doesn't introduce much hassle.